piggybank/src/com/company/Money.java

```diff
@@ -0,0 +1,31 @@
+package com.company;
+
+public abstract class Money {
```

Comment:
Love the way you designed this class. This project technically was asking for abstract classes, but it's clear that this project could be done without any abstract methods. This is the way I would have gone about it, but it would be nice if you made at least one of these methods abstract